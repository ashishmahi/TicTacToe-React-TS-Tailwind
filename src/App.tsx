import React from "react";
import "./App.css";
import "./index.css";

function App() {
  return (
    <div className="App">
      <header className="App-header">
        <p>Lets's Play Tic Tac Toe :)</p>
      </header>
      <button className="py-2 px-4 font-semibold rounded-lg shadow-md text-white bg-red-500 hover:bg-green-700">
        Click me
      </button>
    </div>
  );
}

export default App;
