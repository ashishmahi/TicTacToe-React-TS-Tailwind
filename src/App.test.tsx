import React from "react";
import { render } from "@testing-library/react";
import App from "./App";
describe("App Test", () => {
  it("should render app", () => {
    const { container } = render(<App />);
    expect(container).toMatchSnapshot();
  });
});
