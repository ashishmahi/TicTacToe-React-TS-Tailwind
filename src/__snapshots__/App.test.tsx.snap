// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Test should render app 1`] = `
<div>
  <div
    class="App"
  >
    <header
      class="App-header"
    >
      <p>
        Lets's Play Tic Tac Toe :)
      </p>
    </header>
    <button
      class="py-2 px-4 font-semibold rounded-lg shadow-md text-white bg-red-500 hover:bg-green-700"
    >
      Click me
    </button>
  </div>
</div>
`;
